src/test/ui/suggestions/type-ascription-instead-of-statement-end.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(type_ascription)]
+
+fn main() {
+    println!("test"):
+    0;
```

Comment:
The output in that case is

```rust
error: expected type, found `0`
  --> ../../src/test/ui/suggestions/type-ascription-instead-of-statement-end.rs:15:5
   |
14 |     println!("test"):
   |                     - help: did you mean to use `;` here instead?
15 |     0:
   |     ^ expecting a type here because of type ascription
```